components/resource_control/src/resource_group.rs

```diff
@@ -41,14 +41,9 @@ pub struct ResourceGroupManager {
 impl ResourceGroupManager {
     fn get_ru_setting(rg: &ResourceGroup, is_read: bool) -> u64 {
         match (rg.get_mode(), is_read) {
-            (GroupMode::RuMode, true) => rg
+            (GroupMode::RuMode, _) => rg
```

Comment:
We can remove the parameter `is_read` now

Reply:
keep it to consider `RawMode`